docs/reference/commandline/build.md

```diff
@@ -245,5 +246,9 @@ the command line.
 
 When `docker build` is run with the `--cgroup-parent` option the containers
 used in the build will be run with the [corresponding `docker run`
-flag](/reference/run/#specifying-custom-cgroups).
+flag](/reference/run/#specifying-custom-cgroups). 
+
+When `docker build` is run with the `--ulimit` option the containers used in 
```

Comment:
I'd reword this a little to actually say they'll be run with the ulimit flag? It's not immediately clear to me what actually happens.